src/python/espressomd/CMakeLists.txt

```diff
@@ -42,10 +38,20 @@ add_custom_command(OUTPUT _espresso/myconfig.pxi
                    DEPENDS gen_pxiconfig
                    )
 
+
 include_directories(${PYTHON_INCLUDE_DIRS})
 include_directories(${NUMPY_INCLUDE_DIR})
 
 add_custom_target(espressomd)
+add_custom_command(TARGET espressomd
+        PRE_BUILD
+        COMMAND ${PYTHON_EXECUTABLE} gen_code_info.py ${CMAKE_SOURCE_DIR}/src/features.def ${CMAKE_SOURCE_DIR}/src/python/espressomd/code_info.pyx
```

Comment:
If it is avoidable, you should not create and edit files in source. Is there a reason why the last input parameter isn't `${CMAKE_CURENT_BINARY_DIR}/code_info.pyx` ?